_docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_common.form_validations.inc

```diff
@@ -96,19 +96,23 @@ function check_moderation_state_validations(array &$form, FormStateInterface $fo
  * @param \Drupal\Core\Form\FormStateInterface $form_state
  */
 function add_events_validations(array &$form, FormStateInterface $form_state) {
-  // Basic date comparison.
-  $end_before_start = $form_state->getValue('field_end_date')[0]['value']->getTimeStamp() < $form_state->getValue('field_start_date')[0]['value']->getTimeStamp();
-
-  if ($end_before_start === TRUE) {
-    // Create the error.
-    $form_state->setErrorByName(
-      'field_end_date',
-      t('End Date must be after Start Date')
-    );
+  // Start and End date checks.
+  $end_date = $form_state->getValue('field_end_date')[0]['value'];
+  if (is_object($end_date)) {
```

Comment:
Would it be more specific to use instanceof in this conditional?

Reply:
Yes it would, fixed in commit 267a145